SS2-Project/Assets/Starstorm2/Modules/Pickups/Equipments/Unfinished/AffixKinetic.cs

```diff
@@ -33,7 +87,8 @@ private void FixedUpdate()
                 {
                     return;
                 }
-                if (!body.outOfCombat && timer > 4f)
+
+                if (HasAnyStacks && !CharacterBody.outOfCombat && timer > 4f)
```

Comment:
HasAnyStacks is unecesary. Again, this check only works for the Interfaces that apply to the behaviour. FixedUpdate is not called when the behaviour is disabled (AKA: when Stacks <= 0)